css/index.css

```diff
@@ -69,6 +69,34 @@ p {
 .container {
     width: 800px;
     margin: 0 auto;
+    padding-bottom: 20px;
+}
+
+header.header {
+    display: flex;
+    justify-content: space-between;
+    margin: 20px 0;
+}
+
+header img {
+    width: 20%;
+}
+
+header.header nav.navLinks {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-end;
+    flex-grow: 1;
+}
+
+header.header nav.navLinks a {
+    text-decoration: none;
+    color: black;
+
+}
+
+img.jumbo {
+    margin: 20px 0; 
 }
 
 .top-content {
```

Comment:
Nice work allowing the .box class to style the child boxes.  In the child, you are only overriding the color of each box.